packages/block-library/src/editor.scss

```diff
@@ -58,7 +58,7 @@
 
 @import "./editor-elements.scss";
 
-:root .editor-styles-wrapper {
+:where(html.editor-styles-wrapper) {
```

Comment:
Test according to original PR:

- https://github.com/WordPress/gutenberg/pull/34510